CHANGELOG.md

```diff
@@ -41,6 +41,7 @@
 - #126 (ark-ec) Use `ark_ff::batch_inversion` for point normalization
 - #131, #137 (ark-ff) Speedup `sqrt` on fields when a square root exists. (And slows it down when doesn't exist)
 - #141 (ark-ff) Add `Fp64`
+- #144 (ark-poly) Add serialization for polynomials and evaluations
```

Comment:
This PR is a breaking change, can you move it to the relevant section. (Its added a serialization trait bound to all polynomials)

Reply:
All the polynomial implementations in this repo have now implemented the serialization. I assume there is another project also using polynomial?

Reply:
IDK if there exists another project that implements polynomial, but if they did their implementation is now broken

Reply:
also could the ark-poly readme be updated for this change (https://github.com/arkworks-rs/algebra/tree/master/poly)

Reply:
On the way